models/ChatBox.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+
+const chatSchema = new Schema({
```

Comment:
We already have a chat schema on post model, please remove

Reply:
I double checked and chatted with Alex and it seems this isn't a duplicate and is just for his chat system.